enterprise/admin.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""
+Django admin integration for enterprise app.
```

Comment:
If our tools (pydocstyle, in this case) are forcing us to do the wrong thing (write useless docstrings, in this case), we either need to tweak those tools are abandon them.